src/main/java/org/humancellatlas/ingest/user/UserController.java

```diff
@@ -106,6 +116,17 @@ private Account getCurrentAccount() {
         return account;
     }
 
+    @GetMapping(path = "/list", produces=APPLICATION_JSON_UTF8_VALUE)
```

Comment:
Have you considered using https://www.baeldung.com/spring-security-method-security#method-security?

Reply:
This would be a really nice fix. I'll read about how the decorator works. Thanks for the suggestion!